src/logcollector/read_multiline_indented.c

```diff
@@ -0,0 +1,120 @@
+/* Copyright (C) 2019, Semper Victus LLC
+ * Copyright (C) 2009 Trend Micro Inc.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+/* Read indentend multi line logs */
+
+#include "shared.h"
+#include "logcollector.h"
+
+
+/* Read multi line indented log files */
+void *read_multiline_indented(int pos, int *rc, int drop_it) {
+    size_t str_len = 0;
+    char *p;
+    char str[OS_MAXSTR + 1];
+    char buffer[OS_MAXSTR + 1];
+    /* Zero buffer and str */
+    buffer[0] = '\0';
+    buffer[OS_MAXSTR] = '\0';
+    str[OS_MAXSTR] = '\0';
+    *rc = 0;
+
+    /* Get new entry */
+    while (fgets(str, OS_MAXSTR - OS_LOG_HEADER, logff[pos].fp) != NULL) {
+
+        /* Get buffer size */
+        str_len = strlen(str);
+
+        /* Check str_len size. Very useless, but just to make sure.. */
+        if (str_len >= sizeof(buffer) - 2) {
+            str_len = sizeof(buffer) - 10;
+        }
+
+        /* Get the last occurrence of \n */
+        if ((p = strrchr(str, '\n')) != NULL) {
+            *p = '\0';
+        }
+
+#ifdef WIN32
+        if ((p = strrchr(str, '\r')) != NULL) {
+            *p = '\0';
+        }
+#endif
+        /* Look for empty string */
+        if ((str_len <= 1) || (str[0] == '\r')) {
+            /* Send existing data if any in buffer */
+            if (buffer[0] != '\0') {
+              if (SendMSG(logr_queue, buffer, logff[pos].file, LOCALFILE_MQ) < 0) {
```

Comment:
The other `read_*` files check to make sure `drop_it == 0` before sending to the queue.
I only see instances of `drop_it == 1` for Windows though, so it may not be necessary.
But it would quiet the following build warning:
```
logcollector/read_multiline_indented.c:18:53: warning: unused parameter 'drop_it' [-Wunused-parameter]
void *read_multiline_indented(int pos, int *rc, int drop_it) {
                                                    ^
1 warning generated.
```